tests/Rector/CodeQuality/SimplifyStrposLowerRector/Wrong/wrong.php.inc

```diff
@@ -0,0 +1,6 @@
+<?php declare(strict_types=1);
+
+strpos(strtolower($string), 'find-me');
+
+//$funcName = 'strpos';
+//$funcName(strtolower($string), 'find-me');
```

Comment:
@todo, try to break things

Related issue: https://github.com/rectorphp/rector/issues/646

Should be resolved in every `(string) FunCall->name` call, probably via `FuncCallAnalyzer`